core/src/views/scrollview.rs

```diff
@@ -106,12 +105,12 @@ impl<L: Lens<Target = ScrollData>> ScrollView<L> {
     {
         VStack::new(cx, content)
             .class("scroll_content")
-            .bind(data.clone(), |handle, data| {
-                let data = data.get(handle.cx);
-                let left = ((data.child_x - data.parent_x) * data.scroll_x).round();
-                let top = ((data.child_y - data.parent_y) * data.scroll_y).round();
-                handle.left(Units::Pixels(-left)).top(Units::Pixels(-top));
-            })
+            .left(data.clone().map_shallow(|data| {
```

Comment:
So does this mean that `map_shallow` replaces the need for `.bind()` when using it to set multiple properties?